pkg/universe.dagger.io/go/build.cue

```diff
@@ -46,8 +49,24 @@ import (
 			}
 		}
 		command: {
+			//FIXME: find a better workaround with disjunction
```

Comment:
use the same system as the one in go.#Test.
Using disjunction, and ignoring `package` if `packages` is set.